tests/openwisp2/settings.py

```diff
@@ -74,9 +74,9 @@
 SITE_ID = 1
 
 STATICFILES_FINDERS = [
+    'openwisp_utils.staticfiles.DependencyFinder',
```

Comment:
Since, `django_loci` is not present in INSTALLED_APPS, the static files finder give preference to `leaflet.draw.i18n.js` file contained in django-leaflet package.

If we move this workaround to django-loci, then we will also need to alter the order of `STATICFILES_FINDERS`. This will make the staticfile backend to give preference to files included in EXTENDED_APPS. This may have unwanted outcome (e.g. when a file of EXTENDED_APP is overridden in INSTALLED_APPS.